src/trove_classifiers/__init__.py

```diff
@@ -310,6 +310,7 @@
     "License :: OSI Approved :: Mozilla Public License 1.1 (MPL 1.1)",
     "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
     "License :: OSI Approved :: Mulan Permissive Software License v2 (MulanPSL-2.0)",
+    "License :: OSI Approved :: NASA Open Source Agreement (NOSA)",
```

Comment:
```suggestion
    "License :: OSI Approved :: NASA Open Source Agreement v1.3 (NASA-1.3)",
```
Per https://opensource.org/license/nasa1-3-php, OSI calls this "NASA Open Source Agreement v1.3" and the SPDX Identifier is `NASA-1.3`